tutorials/Native_distributions_and_local_execution/README.md

```diff
@@ -408,3 +409,79 @@ compose.desktop {
     }
 }
 ```
+
+## Customizing Info.plist on macOS
+
+We aim to support important platform-specific customization use-cases via declarative DSL.
+However, the provided DSL is not enough sometimes. If you need to specify `Info.plist`
+values, that are not modeled in the DSL, you can work around by specifying a piece
+of raw XML, that will be appended to the application's `Info.plist`.
+
+### Example: deep linking into macOS apps
+
+1. Specify a custom URL scheme:
+``` kotlin
+// build.gradle.kts
+compose.desktop {
+    application {
+        mainClass = "MainKt"
+        nativeDistributions {
+            targetFormats(TargetFormat.Dmg)
+            packageName = "Deep Linking Example App"
+            macOS {
+                bundleID = "org.jetbrains.compose.examples.deeplinking"
+                infoPlist {
+                    extraKeysRawXml = macExtraPlistKeys
+                }
+            }
+        }
+    }
+}
+
+val macExtraPlistKeys: String
+    get() = """
+      <key>CFBundleURLTypes</key>
+      <array>
+        <dict>
+          <key>CFBundleURLName</key>
+          <string>Example deep link</string>
+          <key>CFBundleURLSchemes</key>
+          <array>
```

Comment:
How one defines supported scheme for other platforms, i.e. Windows or Linux?
Per https://stackoverflow.com/questions/50788945/how-to-make-windows-java-based-app-handle-deep-linking seems that we need to patch manifest or registry.

We could merge this change, as it fixes macOS, but could think of other platforms support.

Reply:
I've created a separate issue https://github.com/JetBrains/compose-jb/issues/855